### DISCUSSION TITLE
Correct citation for EGSnrc?

### DISCUSSION THREAD

**Discussion by ojalaj** [nrc-cnrc/EGSnrc#691] [Questions] [answered]
In https://github.com/nrc-cnrc/EGSnrc/wiki/FAQ there is:

I Kawrakow, E Mainegra-Hing, DWO Rogers, F Tessier, BRB Walters. 
The EGSnrc Code System: Monte Carlo simulation of electron and photon transport. 
Technical Report PIRS-701, National Research Council Canada (2017).

However, to my knowledge, https://nrc-cnrc.github.io/EGSnrc/doc/pirs701-egsnrc.pdf is updated more or less annually, so should one always cite the document version with year that matches the EGSnrc release they have used - e.g., for EGSnrc 2018 release it would the above, but 2018?

Or do you want users to use only one form of citation, so that it would get the number of citations it deserves?

**Comment by ftessier** [comment, marked as answer]
The recommended way to cite EGSnrc is now detailed in the wiki page [Citing EGSnrc](https://github.com/nrc-cnrc/EGSnrc/wiki/Citing-EGSnrc).

**Reply by rtownson** [reply]
The guide doesn't mention user manuals - how should a user cite PIRS-701 or 898 when they want to refer to how a particular component is described in the manual?
